app/views/handlers/individual_response.py

```diff
@@ -225,16 +235,38 @@ def __init__(
         )
 
     def handle_get(self):
-        if self._request_args.get("journey") == "change":
+        if self._return_to_hub:
             previous_location_url = url_for(
-                "individual_response.get_individual_response_change",
-                list_item_id=self._list_item_id,
+                "individual_response.get_individual_response_who",
+                return_to=self._return_to,
             )
-        else:
+        elif self._request_args.get("journey") == "change":
             previous_location_url = url_for(
-                "individual_response.request_individual_response",
+                "individual_response.get_individual_response_change",
                 list_item_id=self._list_item_id,
             )
+        else:
+            self._list_name = self._schema.get_individual_response_list()
+            list_model = self._questionnaire_store.list_store[self._list_name]
+            non_primary_members = 0
+
+            for list_item in list_model.items:
+                if list_item != list_model.primary_person:
+                    non_primary_members += 1
+
+            if non_primary_members > 1:
```

Comment:
This isn't needed if we persist the request args, because that would tell you if you came from the hub.

Reply:
This is able to be edited though?

Reply:
Do you mean the query param can be edited?. If so, yes but that shouldn't matter as long as it is one of the ones we are expecting, as we need to handle that scenario anyway.

Reply:
As discussed offline, it's likely this logic doesn't belong in this handler if we skip over it when there is only a single member.